src/ddprof_worker.cc

```diff
@@ -390,10 +396,14 @@ DDRes ddprof_worker_init(DDProfContext *ctx) {
         ddprof_exporter_new(ctx->worker_ctx.user_tags, ctx->worker_ctx.exp[0]));
     DDRES_CHECK_FWD(
         ddprof_exporter_new(ctx->worker_ctx.user_tags, ctx->worker_ctx.exp[1]));
+
+    // The export profiles non-optionally have the default SampleType of
+    // cpu-time. This should maybe change when we have more than one
+    // customer-facing type.
     DDRES_CHECK_FWD(pprof_create_profile(ctx->worker_ctx.pprof[0],
-                                         ctx->watchers, ctx->num_watchers));
+                                         DDPROF_PWT_CPU_NANOS, 999));
```

Comment:
Why `999` ?

Reply:
Just to avoid round numbers to prevent sampling at a multiple of a periodic activity.

Reply:
That's a fair point.  I redid it in the current API, but I'm still working out whether it makes sense for tests.  🤔